### ISSUE TITLE
Markdown parser breaks some Japanese sentences

### ISSUE THREAD

**Issue by imbsky** [prettier/prettier#5368]
**Prettier 1.15.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEhVeUOYMgghkCGhh0s0CIMgGgyCBDCYOoMg+gyACRuoNMMgxwyA-DA44Mc2+gZgz4WDRcoAYMgRQZA9gyAMKMBthmS6BohkBADAB0oIADQgIABxgBLaAGdkoAIYAnIxADuABWMI9KAwDcImgCbKQAIyMGwAazgwAZTVvTSgAc2QYIwBXOBUACxgAWwAbAHV4zXgdYLA4AJsszXssgE9kcB09FVCdOCMYCy8wpINkADMDFLqVACsdAA8AIS9ffwCDJLgAGVC4Dq6ekH6BgNCwlLgARWiIeAXuuJBgozqjCtajH2dzRRU1Uzq0rzUKh7gz+3mVIzgAR2iml+TQMLTaSE6hxUdSSmgOSx0602Oz28yQUViKhgBncaRcMHiyAATFivJoUusAMIQJKtCpQaDfEDROoAFRxtkhSweoRgeOcBOQAEYiQAGAC+4qAA)

```js
module.exports = {
  printWidth: 120,
  semi: false,
  singleQuote: true
}
```

**Input:**

```md
何でも薄暗いじめじめした所でニャーニャー泣いていた事だけは記憶している。
```

**Output:**

```md
何でも薄暗いじめじめした所でニャ ー ニャ ー 泣いていた事だけは記憶している。
```

**Expected behavior:**

> ー

This is a short, dash‐like character signifying that the preceding syllable should be lengthened in Japanese. But, in this example, there are unnecessary spaces before and after that character. And these spaces should not be inserted. Because it breaks some Japanese sentences.

**Comment by ikatyang** [comment]
The fix is available in [`1.15.2`](https://github.com/prettier/prettier/blob/master/CHANGELOG.md#1152).

**Comment by imbsky** [comment]
Thanks, you are doing god's work!
